can-connect.md

```diff
@@ -329,7 +329,7 @@ Todo.List = DefineList.extend({
 
 var todoConnection = connect([
     require("can-connect/data/url/url"),
-    require("can-connect/can/map/map/map"),
+    require("can-connect/can/map/"),
```

Comment:
I think it should be `can-connect/can/map/map`. We don’t drop the module name from the end in the CanJS docs because that’s StealJS-specific.